extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/AccumuloIndexingConfiguration.java

```diff
@@ -287,15 +286,19 @@ public void setAccumuloTemporalPredicates(String[] predicates) {
     public String[] getAccumuloTemporalPredicates() {
         return getStrings(ConfigUtils.TEMPORAL_PREDICATES_LIST);
     }
-    
-    private static Set<RyaURI> getPropURIFromStrings(String ... props) {
-        Set<RyaURI> properties = new HashSet<>();
-        for(String prop: props) {
-            properties.add(new RyaURI(prop));
+
+    private static Set<RyaURI> getPropURIFromStrings(final String... props) {
+        final Set<RyaURI> properties = new HashSet<>();
+        if (props != null) {
+            for(final String prop : props) {
+                if (prop != null) {
```

Comment:
Do we want/need an empty string check here?  Do we need/have a unit test to exercise this?

Reply:
Added a check for if the prop is blank.  There are unit tests for comma-separated values.